dev-cluster/Dockerfile

```diff
@@ -0,0 +1,83 @@
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not
+# use this file except in compliance with the License. You may obtain a copy of
+# the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations under
+# the License.
+
+FROM debian:jessie
+
+MAINTAINER CouchDB Developers dev@couchdb.apache.org
+
+ENV COUCHDB_VERSION master
+
+RUN groupadd -r couchdb && useradd -d /usr/src/couchdb -g couchdb couchdb
+
+# download dependencies
+RUN apt-get update -y -qq && apt-get install -y --no-install-recommends \
+    apt-transport-https \
+    build-essential \
+    ca-certificates \
+    curl \
+    default-jdk \
+    erlang-dev \
+    erlang-nox \
+    git \
+    haproxy \
+    libcurl4-openssl-dev \
+    libicu-dev \
+    libmozjs185-dev \
+    libwxgtk3.0 \
+    openssl \
+    pkg-config \
+    python \
+    python-sphinx \
+    texinfo \
+    texlive-base \
+    texlive-fonts-extra \
+    texlive-fonts-recommended \
+    texlive-latex-extra \
+ && curl -s https://deb.nodesource.com/gpgkey/nodesource.gpg.key | apt-key add - \
+ && echo 'deb https://deb.nodesource.com/node_6.x jessie main' > /etc/apt/sources.list.d/nodesource.list \
```

Comment:
Node 6 is end of life now. Could you update this to use node 8 which is the current LTS.

Reply:
Certainly. That was a straight copy/paste from the old Dockerfile but happy to upgrade it as part of this PR.

Reply:
@garrensmith The README in Fauxton still says we require "at least Node 6 and npm 3", is that no longer true? If not we'll need to change the build for non-dev instances, too.

Reply:
Its fine to use node 6. I recall you asking me to try to use node 6 for as long as possible so I have intentionally made sure it still works on 6. But since this is a dev image it would be better if we were using a more recent version of Node.

Reply:
@garrensmith out of curiosity where do I go to find the node release cycle? I tried https://github.com/nodejs/Release but I couldn't match it up with your comment here.

Reply:
@kocolosk my guess is that he means it moves to maintenance LTS in a couple of months, i.e. no more improvements other than security patches...and it's probably fair to say that it's not getting a lot of actual changes these days, either... ;)